.github/ISSUE_TEMPLATE/pre-release-demos-nightly.md

```diff
@@ -0,0 +1,100 @@
+---
+name: Pre-Release Demo Upgrade Testing from Stable to Nightly
+about: This template can be used to track the upgrade testing of demos from stable to nightly leading up to the next Stackable release
+title: "chore(tracking): Test demos on nightly versions"
+labels: ['epic']
+assignees: ''
+---
+
+<!--
+    Make sure to update the link in '.github/ISSUE_TEMPLATE/release.md' when
+    you change the front matter above.
+-->
+
+<!--
+    DO NOT REMOVE THIS COMMENT. It is intended for people who might copy/paste from the previous release issue.
+    This was created by an issue template: https://github.com/stackabletech/issues/issues/new/choose.
+-->
+
+## Pre-Release Demo Testing on Nightly
+
+Part of <https://github.com/stackabletech/issues/issues/TRACKING_ISSUE>
+
+For each demo, run the following commands:
+
+> [!NOTE]
+> Record any issues or anomalies during the process in a comment on this issue.
+> Eg:
+> ```
+> :green_circle: **airflow-scheduled-job**
+>
+> The CRD had been updated and I needed to change the following in the manifest:
+> ...
+> ```
+
+```shell
```

Comment:
**nit(non-blocker)**: this should be `sh` (or `bash` if necessary).

~~~suggestion
```sh
~~~

_you don't need to change it now, but I probably will when I next see it_

Reply:
Decided to leave as is.